packages/dev/core/src/Materials/greasedLinePluginMaterial.ts

```diff
@@ -148,38 +161,73 @@ export class GreasedLinePluginMaterial extends MaterialPluginBase {
      */
     public static readonly GREASED_LINE_MATERIAL_NAME = "GreasedLinePluginMaterial";
 
+    /**
+     * Default line color for newly created lines
+     */
+    public static DEFAULT_COLOR = Color3.White();
+    private static _EmptyColorsTexture: BaseTexture;
+
     private _colorsTexture?: RawTexture;
 
-    private _dashArray = 0;
+    private _visibility: number;
+    private _useDash: boolean;
+    private _dashCount: number;
+    private _dashRatio: number;
+    private _dashOffset: number;
+    private _dashArray: number;
+    private _width: number;
+    private _sizeAttenuation: boolean;
+    private _color: Color3 | undefined;
+    private _colorMode: GreasedLineMeshColorMode;
+    private _colors: Nullable<Color3[]> | undefined;
+    private _useColors: boolean;
+    private _colorDistributionType: GreasedLineMeshColorDistributionType;
+    private _colorsSampling: number;
+    private _resolution: Vector2;
 
     private _options: GreasedLineMaterialOptions;
 
     private _engine: Engine;
 
-    private static _EmptyColorsTexture: BaseTexture;
-
-    constructor(material: Material, private _scene: Scene, options: GreasedLineMaterialOptions) {
-        options = options ?? {};
+    constructor(material: Material, private _scene: Scene, options?: GreasedLineMaterialOptions) {
+        options = options || {
+            color: GreasedLinePluginMaterial.DEFAULT_COLOR,
+        };
 
         const defines = new MaterialGreasedLineDefines();
-        defines.GREASED_LINE_HAS_COLOR = !!options.color;
+        defines.GREASED_LINE_HAS_COLOR = !!options.color ?? false;

```

Comment:
You don't need the `?? false` part, `!!` will be fine as it converts to boolean.

Reply:
Yeah, I changed this to something else then removed some code from this line and forgot to remove the `?? false`... Removed.